chat-plugins/modlog.js

```diff
@@ -280,15 +318,30 @@ exports.commands = {
 		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && targetRoom.isPrivate !== true));
 		// Let's check the number of lines to retrieve or if it's a word instead
 		let lines = 0;
+		let more = false;
+		let moreIncrement = 0;
+		if (target.includes('LINE INCREMENT=')) {
```

Comment:
So, should I document that you can specify the lines with ``lines=`` in the help command?

Until now, I was under the impression that this all was going to be used in the UI exclusively